### DISCUSSION TITLE
GEM

### DISCUSSION THREAD

**Discussion by MeanXBeam** [iptv-org/community#539] [Q&A]
Hello i was wondering if there was anyway to change the subtitles to english or if i could find  english subtitled one.

**Comment by Jitterdoomer** [comment]
> Hello i was wondering if there was anyway to change the subtitles to english or if i could find  english subtitled one.

I'm sorry for late answer, the one on Stream-Link.org has the English subtitled one but the announcer is Mandarin Chinese.
